pfsspy/utils.py

```diff
@@ -88,7 +88,7 @@ def load_adapt(adapt_path):
     return adaptMapSequence
 
 
-def carr_cea_wcs_header(dtime, shape):
+def carr_cea_wcs_header(dtime, shape, map_center_longitude=0*u.deg):
```

Comment:
```suggestion
def carr_cea_wcs_header(dtime, shape, *, map_center_longitude=0*u.deg):
```
I think making the argument keyword only makes sense here.